src/Pact/Native.hs

```diff
@@ -218,6 +220,22 @@ formatDef =
                   (zip (V.toList es) (tail parts))
     format i as = argsError i as
 
+strToListDef :: NativeDef
+strToListDef = defGasRNative "str-to-list" strToList
+  (funType (TyList tTyString) [("str", tTyString)] )
+  [ "(str-to-list \"hello\")"
+  , "(concat (map (+ \" \") (str-to-list \"abcde\")))"
+  ]
+  "Takes STR and returns a list of single character strings"
+
+concatDef :: NativeDef
+concatDef = defGasRNative "concat" concat'
+  (funType tTyString [("str-list", TyList tTyString)] )
+  [ "(concat [\"k\" \"d\" \"a\"])"
+  , "(concat (map (+ \" \") (str-to-list \"abcde\")))"
+  ]
+  "Takes STR and returns a list of single character strings"
```

Comment:
I don't think the documentation for `concat` is correct. (It is also wrong in the docs, but that should be corrected once you rerun tests).

Reply:
HA! Good catch